### PR TITLE
Update the release process for community release to include release candidate versioning

### PR DESCRIPTION
**Pull request by maniarathi**

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,5 +1,16 @@
 [bumpversion]
 current_version = 0.16.0
+commit = True
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?:-(?P<prerel>rc)\.(?P<prerelversion>\d+))?
```
Comment (MDunitz) [inline .bumpversion.cfg:4]: is this also from the semvar site? could you add a comment about what its doing?

Comment (maniarathi) [reply .bumpversion.cfg:7]: Nope, this is my regex. I added an explanation.

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,5 +1,19 @@
 [bumpversion]
 current_version = 0.16.0
+commit = True
+# The below regex details an acceptable version number by naming the groups (major, minor, patch, prerel, and
+# prerelversion) and also specifying the valid values for each group (positive integers, `\d+`,  for major, minor,
```
Comment (bmccandless) [inline .bumpversion.cfg:5]: comment should just be "integers", not "positive integers".  e.g. 0.16.1

Comment (maniarathi) [reply .bumpversion.cfg]: Done!

**Pull request merged**